components/formats-gpl/src/loci/formats/in/CellWorxReader.java

```diff
@@ -711,14 +714,16 @@ private IFormatReader getReader(String file)
       pnl = new MetamorphReader();
     }
 
-    IMetadata metadata;
-    try{
-      metadata = service.createOMEXMLMetadata();
-    }
-    catch (ServiceException exc) {
-      throw new FormatException("Could not create OME-XML store.", exc);
+    if (omexml) {
+      IMetadata metadata;
+      try {
```

Comment:
Arguably, yes, but whether it should remain that way is up for debate.  Either way, this is part of the decoupling discussion, and completely outside the scope of what this PR is intended to accomplish.